tools/dependency/release_dates.py

```diff
@@ -132,19 +185,26 @@ def verify_and_print_release_dates(repository_locations, github_instance):
 
 
 if __name__ == '__main__':
-    if len(sys.argv) != 2:
+    if len(sys.argv) != 3:
         print('Usage: %s <path to repository_locations.bzl>' % sys.argv[0])
         sys.exit(1)
+    # parsing location and github_action flag with argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument('location', type=str)
+    parser.add_argument('cron', default=False)
```

Comment:
can this have a type boolean and avoid the comparison on line 201? (e.g. just write cron_action = args.cron?)

Reply:
Yea !!